### ISSUE TITLE
App Directory - Design Guidelines

### ISSUE THREAD

**Issue by careyotto** [department-of-veterans-affairs/va.gov-team#9238]
# App Directory

## Use case
Veteran may come here from help links on _Connected Apps FAQ_ or _Connected Apps_ screens, or through search. They are looking for third-party apps they can connect their VA data to. Third-party apps use Lighthouse OAuth APIs.
<br>

## Designs and Prototype
_Designs created in Figma. Contact @careyotto (also on DSVA Slack) for editor access._

**Figma prototypes:**
- [Desktop prototype](https://www.figma.com/proto/Oi4PR8zuoUwwMB2qbQnWtV/VA.gov-App-directory?node-id=89%3A1796&viewport=1869%2C4131%2C0.25&scaling=min-zoom) 
- [Mobile prototype](https://www.figma.com/proto/Oi4PR8zuoUwwMB2qbQnWtV/VA.gov---App-directory?node-id=147%3A272&viewport=1869%2C4131%2C0.25&scaling=min-zoom)
- [Related Github issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7290)
- [Connected Apps - same custom component being built by Auth. Exp.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Design/design-specs/connected-apps.md)

### Description 
This is the third-party app directory. It is hosted through Public Websites and is a product of the VA Lighthouse API team. If you have questions, please contact @careyotto @mshea0606 

The app "widget" is a custom build and can leverage the custom code Authenticated Experience team is building for Profile 2.0 Connected Apps. [(See connected-apps.md)](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Design/design-specs/connected-apps.md)

The logo and app name are pulled in from Okta. To see examples of how this is built, look at Staging > Connected Accounts with test user 1.

The privacy policy links, terms of service links, category, device/platform, and app description are new content and will be linked to here once it's available. If you have any questions, contact @careyotto 
### Content
```
Home  ›  App Directory

# App Directory
Below, you’ll find a list of third-party websites and applications you can link to your personal VA data, like health or service records. Third-party services are created by developers that aren’t VA. It is never required to use them.

[BEGIN filter box on gray bkg with 2 dropdowns. Include 32px of space between clickable components on mobile.]
Choose a device or platform:
[Dropdown] ** All devices **

Choose a service category:
[Dropdown] **  All categories ** 

[Default button] Filter

[END filter box]

Showing 1 – 10 of 51 results for **"all devices"** **"all categories"**

[BEGIN APP WIDGET]
[Logo brought in from Okta - 45x45 pixels]
[App Name brought in from Okta]
[Category] app available for [platform/device]
[Secondary button - opens app's website in a new browser tab] Find app 
[Additional info component - expands widget to show more detail] Learn more about [App Name]
[END APP WIDGET]
[BEGIN APP WIDGET EXTRA CONTENT WHEN EXPANDED]
[1 pixel hairline #D6D7D9]
**About this app:**
[Description copy up to 415 characters]

**[App Name] asks for:**
- [Scope brought in from Okta]
- [Scope brought in from Okta]
- [Scope brought in from Okta]
- [Scope brought in from Okta]
- [Continue as necessary...]

** More information:**
[Link - new browser tab to app's privacy policy] View the privacy policy
[Link - new browser tab to terms of service] View terms of service
[Link - sends email to *api@va.gov*  with subject line “Report [App Name] to VA”] Report this app to VA
[END APP WIDGET]
[20 px space]
[APP WIDGET]
[20 px space]
[APP WIDGET]
[...etc. up to 10 display on a single page]

[Pagination]

[Information callout] *Have questions about connected apps?*
Get answers to frequently asked questions about how connected third-party apps work, what types of information they can see, and the benefits of sharing your information.
[Link - same browser window] Go to connected apps FAQs

[hairline]
Last updated [Month DD, YYYY]

[Footer]

### Design
![Dev Guidelines for App Directory](https://user-images.githubusercontent.com/37549106/82271500-ccb13e00-992c-11ea-9409-e5efb22af599.png)
![App Directory - default view - Desktop](https://user-images.githubusercontent.com/37549106/82271505-cfac2e80-992c-11ea-8936-311e60513175.png)
![App Directory - Widget expanded - Desktop](https://user-images.githubusercontent.com/37549106/82271515-d5a20f80-992c-11ea-99dd-ae6fac81f1e3.png)
![App Directory - default view - Mobile](https://user-images.githubusercontent.com/37549106/82271521-d9359680-992c-11ea-8c61-14cdfb43b50f.png)
![App Directory - Widget expanded - Mobile](https://user-images.githubusercontent.com/37549106/82271530-dc308700-992c-11ea-871a-752e7b1e08f6.png)

**Comment by careyotto** [comment]
![Dev Guidelines for App Directory](https://user-images.githubusercontent.com/37549106/82271844-beafed00-992d-11ea-943d-aff08252f0ae.png)
![App Directory - default view - Desktop](https://user-images.githubusercontent.com/37549106/82271846-c2437400-992d-11ea-84ee-380359bb7a20.png)
![App Directory - Widget expanded - Desktop](https://user-images.githubusercontent.com/37549106/82271851-c4a5ce00-992d-11ea-983e-0d7408922047.png)
![App Directory - default view - Mobile](https://user-images.githubusercontent.com/37549106/82271853-c66f9180-992d-11ea-92c0-362b2cd6da03.png)
![App Directory - Widget expanded - Mobile](https://user-images.githubusercontent.com/37549106/82271857-c8d1eb80-992d-11ea-981e-4f6089df4dfd.png)

**Comment by kelsonic** [comment]
This sounds like so much fun to build! I have a few questions:

1. Do we have user stories for this?
1. Are we sure that the user flow is correct? I just want to confirm that we want to push our visitors to these 3rd party connected app pages.
1. Once we have the above ^^, this is the 3rd app that is following the pattern `Find X` (e.g. Yellow Ribbon + Find Forms). What we learned from Find Forms is that having a11y baked into the design drastically helped engineering when it came time to implement it. Could we do the same here and ensure that the design is first audited by a11y to save us back-and-forth time in the future?
1. What is the API endpoint from Lighthouse I should look into to get a list of the connected apps? Is it public or is it behind some sort of authentication I'll need?

Thanks and super excited about this new epic! 🙂

**Comment by kelsonic** [comment]
@stephorkandatx Should we convert this to an epic?

**Comment by lunascoop** [comment]
@kelsonic I think so. I think will include some content recommendations for this page.

**Comment by careyotto** [comment]
@kelsonic You just made my day! I'm glad to hear you are excited about working on this. :)

To answer your questions:
1. User story: As a veteran, I want to find third-party apps I can use to access my data outside of VA.gov.

2. Yes, we're good on flow. Links to this app directory will live on the [Connected Apps page](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Design/design-specs/connected-apps.md) and [FAQ page](https://github.com/department-of-veterans-affairs/va.gov-team/issues/6838). The only way for a Veteran to connect to a third-party app is from the app itself, so we need to send them there to complete those flows. I would recommend links off-site open a new browser tab. 

3. I am unfamiliar with the Yellow Ribbon + Find Forms learnings. Curious to learn more and am open to updating this app directory design to support a11y. @1Copenut reviewed the design during VSP collaboration cycle and suggested adding the "Showing 1-10 results...." copy. Is there more that needs to be done here?

4. `What is the API endpoint from Lighthouse I should look into to get a list of the connected apps?` No API, just a manual list that I will provide in a jiffy. The Lighthouse team is working on creating a database for this, but it won't be ready in time for your work. App logo, app name, and scopes need to be pulled in from our OAuth solution (Okta) like is being done in the current "connected accounts" page on VA.gov. The rest of the info will need to be manually inputed.

We have a check-in mtg with tomorrow, so more updates from may come after that.

**Comment by careyotto** [comment]
@kelsonic Turns out I need to track down a few more app descriptions and links. Once that is done, you'll be able to find them in this directory: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/connected-apps-data

**Comment by kelsonic** [comment]
Sounds great! Thank you 🙂Looking forward to chatting with you tomorrow!

**Comment by careyotto** [comment]
@jenstrickland Would love your eyes on this design, from an accessibility POV. Will follow up via Slack.

**Comment by jenstrickland** [comment]
@kelsonic Thank you! 🥇 

<hr/>

@careyotto If development for this section uses Formation, there may be constraints that interfere with the columns. If y'all can use @erikphansen 's component, that will provide more flexibility.

1. What is the flow order of the content? From the mobile app mockup, I think it is:
    1. icon
    1. App Name
    1. Category app available for platform/device
    1. Find app / Connect / Disconnect
    1. Learn about app show/hide

I ask because I'm thinking about the non-sighted user attempting to use this, and wonder if they would need to Learn about the app before the action. This matters for how the code is structured. I recommend the markup reading order below, and if you want the button to appear as shown in your mockup, I would use CSS to position. Let me know if y'all need me to provide a quick Codepen on how to do this.

**Potential markup**

```html
<h2>Showing 1 through 10 of 51 results for…</h2>
<ol role="list"> <!-- adding role for list because the CSS to remove the bullets will remove the list semantics for assistive technology, and this will add it back -->
  <li role="listitem">
    <h3 id="unique_id01">
      <i aria-hidden="true" role="presentation" class="font-awesome-classes"></i> 
      App Name
    </h3>
    <ul role="list" aria-labelledby="unique_id01">
      <li role="listitem">Category app available for platform/device</li>
      <li role="listitem">Learn about app show/hide</li>
      <li role="listitem">Button</li>
    </ul>
  </li>
  <li role="listitem">
    <h3 id="unique_id02">
      <i aria-hidden="true" role="presentation" class="font-awesome-classes"></i> 
      App Name
    </h3>
    <ul role="list" aria-labelledby="unique_id02">
      <li role="listitem">Category app available for platform/device</li>
      <li role="listitem">Learn about app show/hide</li>
      <li role="listitem">Button</li>
    </ul>
  </li>
<!-- etcetera -->
</ol>
```
<hr/>

2. For the desktop design, be mindful of proximity  the space between where the app name and descriptor text is and then the space between that and the button. If you use an empty toilet paper roll, view your mockup and notice how it may be hard to see the button. For people who use magnification tools, they may not see the button. I think that the line that contains the box may guide the eye to the box, but I would like to validate it. I expressed the same concern with Profile 2.0, so the box was made less wide than it was previously.

<hr/>

3. For the modal, this may be tricky to manage focus. If there is a way to avoid using a modal, I would encourage that. Iirc Profile 2.0 is moving to edit in place rather than modal. Perhaps the confirmation could be done inline? Since you're likely to need to proceed with the modal:
    a. Check with Ryan Thurwell on his latest card patterns for the action and cancel button/link design.
    b. For the Close button X, it will need some screen reader only text to clarify what is available to close. Here's an example:

  **Current code sample**

```html
            <div class="va-modal" id="profile-edit-modal" role="alertdialog">
              <div class="va-modal-inner">
                <button class="va-modal-close" type="button" aria-label="Close this modal">
                  <i class="fas fa-times-circle" aria-hidden="true"></i>
                </button>
                <div class="va-modal-body">
                  <div role="document">
                    <div class="">
                      <h3>Edit mailing address</h3>
                      <div>
                        <form class="rjsf" novalidate="">
<!-- etcetera -->
```

  **Recommended code sample**

```html
            <div class="va-modal" id="profile-edit-modal" role="alertdialog">
              <div class="va-modal-inner">
                <button class="va-modal-close" type="button" aria-label="Close Edit mailing address modal">
                  <i class="fas fa-times-circle" aria-hidden="true" role="presentation"></i>
                </button>
                <div class="va-modal-body">
                  <div role="document">
                    <div class="">
                      <h3>Edit mailing address</h3>
                      <div>
                        <form class="rjsf" novalidate="">
<!-- etcetera -->
```
<hr/>

4. Processing form button — this button was problematic for the Authenticated Experience Address Validation work, and there is a known bug in the design system component. The bug is documented in https://github.com/department-of-veterans-affairs/va.gov-team/issues/5865 and https://github.com/department-of-veterans-affairs/va.gov-team/issues/7324. Be sure to have this validated with JAWS + IE11 (this is the combination used by the VA 508 Office, so we need to make sure stuff works). 

<hr/>

5. I checked the design review comments and didn't see this, so I thought I'd note it here, although I suspect the dev would add the padding. I can be a wee too detaily — after being a project manager, editor, and production artist for print where corrections are costly.

![Screen Shot 2020-05-21 at 1 20 04 PM](https://user-images.githubusercontent.com/57469/82586664-eee1d080-9b65-11ea-878c-45e0518130e5.png)

<hr/>

I think that's it. Please let me know if there are any questions! 😺

**Comment by careyotto** [comment]
Thank you @jenstrickland ! Super fast feedback, wow!

Re: modal, there aren't any modals in the App Directory, just Connected Accounts.
Re: padding on blue tout at bottom of screen -- good catch! The lack of it is a mistake on my part, not intentional.

Happy to align with whatever modifications Auth. Exp. is using for this component.

**Comment by kelsonic** [comment]
May I get a list of all potential `device`s and `category`(ies) that we would show in the dropdowns when you have time @careyotto? 🙂

**Comment by kelsonic** [comment]
Also, what are we expecting exactly to do with the `Report this app to the VA` link? Would it be a link or would it act like a button that does something when it's clicked?

**Comment by careyotto** [comment]
@kelsonic At the moment, we only have 4-5 apps for the directory. All the details you're asking for can be found here: https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/connected-apps-data

Currently 1 category: Health
Currently 3 platform/devices: iOS, Android, web browsers

Report this app to VA link sends email to *api@va.gov*  with subject line “Report [App Name] to VA”, as documented above.

**Comment by kelsonic** [comment]
@careyotto Perfect, thank you! 🔥

**Comment by careyotto** [comment]
Note, apps can belong to more than one platform/device category.

**Comment by kelsonic** [comment]
Could you provide me with the default icon in case we don't have that for an app? I'm seeing a cog there, not sure if we want to use the one from font awesome 🙂

**Comment by careyotto** [comment]
The default icon is automatically generated from Okta.

**Comment by kelsonic** [comment]
The Okta endpoint is behind auth unfortunately, so I don't think we'll be able to use that for a non-authenticated app: https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/api-reference/#/profile/getConnectedApplications

**Comment by kelsonic** [comment]
^^ assuming that's the okta endpoint you were referring to of course! 🙂

**Comment by stephorkandatx** [comment]
DEPO owners for PW and Auth team were able to review the original request in hosting content related to Connected Apps. The primary objective was to provide simple FAQs to help the VA.gov users with how to disconnect third-party apps, as well general information about what Connected Apps are. These can and will be accomplished through the FAQs content being generated.

We have agreed to not implement the Apps Directory at this time, as it will not be maintainable and sustainable dynamically. If there is a dynamic way in the future, we can review and prioritize the request.

Before we publish a list of apps on the unauthenticated page, we will need to confirm that they have been vetted and approved. Would it be possible to add that information to this page  https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/connected-apps-data/App%20Directory%20-%20Production%20List.csv or is this information already available elsewhere?

**Comment by careyotto** [comment]
@stephorkandatx All apps on that list have been vetted and approved by VA before receiving production access to our OAuth APIs. Let's add this item – and the meeting/decision you mention above – to our talking points for tomorrow's 12:30pm Eastern synch.

**Comment by Samara-Strauss** [comment]
Hey guys, I want to apologize around some of the swirl that happened with this feature. I think it was my fault.

At some point, Carey and I had talked about how it would be nice to have a list of apps people could connect to give them more context about this feature if they were on the VA.gov profile, which I manage as part of the authenticated experience portfolio. That conversation was happening at the same time and in the same spaces as the conversation about the connected apps FAQ. The connected apps FAQ **is** a requirement for us to have for the connected apps section in the new profile 2.0, but the list of apps — what is now being called the apps directory — is not a requirement. However, I did not make that clear to Carey and Carey thought the apps directory was a requirement like the FAQ. This is my fault and I apologize. I know it's caused a lot of back and forth and confusion, and that's on me.

Going forward, the only requirement from the authenticated experience team is that the FAQ for connected apps gets done before we launch profile 2.0 (which is still off by a few sprints). I know that work is already well under way in #6838.

However, the apps directory is not a requirement for the authenticated experience team or the profile 2.0 launch, so whether it is (1) possible or (2) whether it can get built now, later, or ever is up to @jenniferlee-dsva and the public websites team. I won't tell y'all what to do here but I do want to be clear that it's not a requirement from me or the authenticated experience team. Again, I apologize that this was not clear before and I'm sorry that I was not more involved in conversations to catch this sooner.

**Comment by jenniferlee-dsva** [comment]
Thanks for the clarification @Samara-Strauss. 

@stephorkandatx - let's make sure the directory part of this request is separated from the user help content about connecting/disconnecting apps. I don't think we should create a directory on the PW site that can't be dynamically maintained. If at some point, it can be, that might be the right time to do it. Or alternatively, the directory just becomes a list of links taking people to the google and apple app stores....  But TBH, I'm just not convinced there's even a need for that at the moment. 
Will defer to you and @DanielleThierryUSDSVA to recommend the better PW experience for Veterans.

**Comment by Samara-Strauss** [comment]
> Will defer to you and @DanielleThierryUSDSVA to recommend the better PW experience for Veterans.

Again, I want to reiterate a list of apps is not required at all, but what I had originally pictured was just an FAQ that said "what apps can I connect?" and a non-dynamic list of apps. I realize now, though, that this might be hard to maintain if the list of apps is growing rapidly, BUT we could potentially still consider a simple solution like that if the list of apps grows pretty slowly and would only need periodic updates every few months (which @careyotto would be responsible for initiating).

That's my one idea, but I don't think we need anything more complicated than that. If even an FAQ that addresses a list of apps isn't feasible for whatever reason (eg. too hard to maintain) then that is OK and we can do without it.

**Comment by jenniferlee-dsva** [comment]
Hi @careyotto et al: 

I know this has been on hold until further clarification from the PW team. Confirming here for everyone. 

**1/ Apps directory:** 
We are not creating an apps directory at this time on the unauthenticated/public website. At some point in the future, when there are many apps and the information can be maintained via API, we can regroup.

**2/ Connected apps FAQs recommendation:** 
After reviewing the FAQ information in more depth, and consulting with the PW and Platform content and IA teams, we recommend the approach below. Peggy and Danielle will work with you and @Samara-Strauss to help finalize content. 
@Samara-Strauss and I support this content strategy recommendation. 

- Include the most directly relevant help-related information in the authenticated experience, in the connected or empty state as relevant. Since the information is relatively minimal and highly contextual, we recommend this to give the user the information they need to make decisions right on the page, and if appropriate, linking the user out to see more general help info.  We recommend including a simple list of available apps here as well, framed in the connected state as “Other apps you may want to connect to” and in the empty state as “Apps you can connect to” (specific wording tbd).
- Include a few general questions as a “connected apps” section within the existing Sign in FAQs page. This will allow users who aren’t signed in to find this general content via search. In the future, all of this will be great content to move into the VA.gov learning center, which will be designed to house help-related content and other resource-focused content. 
- For developer questions: Consider developer.va.gov as a place to house the question about accessing the Lighthouse API for developers. 

FYSA @stephorkandatx @kelsonic @Samara-Strauss @tressaellen @DanielleThierryUSDSVA @peggygannon @mnorthuis @lunascoop

**Comment by Samara-Strauss** [comment]
> Include the most directly relevant help-related information in the authenticated experience, in the connected or empty state as relevant. 

As Jen mentioned, I am OK with this approach. **However**, in order to avoid the user profile from becoming a lengthy FAQ, we'll have to work together to pare down the FAQ from what it is now and only include whatever is most relevant and most needed. We don't have a separate FAQ page/section for authenticated tools, and while maybe that may be something we want to consider in the future, it would be out of scope fo this particular project. So, that leaves us putting things contextually in the Connected Apps section of the Profile, and I want to make sure we curate that with care.

**Comment by careyotto** [comment]
Hi @jenniferlee-dsva and @Samara-Strauss -- This new approach to the FAQs is a big change from what our teams agreed upon. 

As a reminder, these Connected Apps FAQs were originally designed to be available from an unauthenticated experience over a year ago. The original intention was to link to them from the authentication screen a Veteran sees when they are trying to connect to a third-party app outside of VA.gov.

[*Original ticket from 2018 where it was determined that these FAQs were beyond the scope of sign-in FAQs. ](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15901)

We were never able to implement this feature on our end because we were waiting on the FAQ page to be built. (Copy and URL had been approved by VSA.)

In the meantime, we discovered a second use case for having FAQs linked to from the Profile 2.0. So we picked this up again, updated the copy based on real Veteran questions our support team gets, and had the concept and copy vetted and approved by both VSP and VSA. This has been months in the making and vetted many times by at least 3 VA teams (VSP, PW, Lighthouse). I'm concerned by the sudden change of approach.

Can we discuss this?

Looping in my Product Owners @afichter-va @davem-va and our new Product Manager @markgreenburg for their input and visibility as well.

**Comment by jenniferlee-dsva** [comment]
Hi @careyotto - AS PO of the public website team, I'm already in contact with Andrew Fichter about other things, so I'll reach out to him about this.  @davem-va  @afichter-va

**Comment by Samara-Strauss** [comment]
Hey @careyotto — First, I want to say that I get this must be really frustrating. I know a lot has changed in the last few days when it seemed like things were rolling along smoothly. 

> This new approach to the FAQs is a big change from what our teams agreed upon.
> 
> As a reminder, these Connected Apps FAQs were originally designed to be available from an unauthenticated experience over a year ago. The original intention was to link to them from the authentication screen a Veteran sees when they are trying to connect to a third-party app outside of VA.gov.

Totally, this was definitely the original approach. However, from what I understand, the recommendations from the public websites team has changed. That happens sometimes and at least from a user profile perspective, I am OK with this.

Also, for the most part, everyone who worked on the last iteration of this is no longer on this team/contract, as our old contract ended last summer. This may be part of why the recommendation from 18 months ago is different from now. However, we are lucky that @DanielleThierryUSDSVA is still here — she wrote the original FAQ content as a contractor and now is part of the VA DEPO team.  She'd be the best person to provide context on what happened then and why what they're recommending now is a better approach.

> In the meantime, we discovered a second use case for having FAQs linked to from the Profile 2.0. So we picked this up again, updated the copy based on real Veteran questions our support team gets, and had the concept and copy vetted and approved by both VSP and VSA. 

Yes, agreed! This is important to the profile work, and I'm grateful that you've been managing it. 

As I understand, the recommendation from PW is to have some of this content directly in-line in the profile. While I don't want the profile to turn into a long FAQ itself, I do think there is probably room to have relevant in-line content on connected apps, especially if people have no apps connected. Actually, that was the big reason we wanted FAQs in the first place – if someone goes to that section of the profile and they have nothing connected, how do we let them know what connected apps are and how they might use them? If we can provide that context in-line, I actually think this is a great recommendation and better user experience!

While this doesn't address some things like developer content (obviously not appropriate for the profile) or content on trouble-shooting, this at least rectifies the profile use case, which was the biggest reason for the FAQs at this point.

> This has been months in the making and vetted many times by at least 3 VA teams (VSP, PW, Lighthouse). I'm concerned by the sudden change of approach.

At least for my part, I was OK with the original approach to the FAQs because I wasn't aware there'd be any changes or a different recommendation from the PW team. But now that there is a different approach, I trust this recommendation and think we should follow it.

That being said, I do completely understand your frustration around some of the communication on this issue. This issue has highlighted how our team (DEPO/VSA/VSP) really needs to work on how we work with teams that don't have a dedicated DEPO contact. I don't have an answer or explanation, but do want to acknowledge that we could have done a better job at communicating across teams instead of throwing things over the wall to each other.

**Comment by jenniferlee-dsva** [comment]
All - @Samara-Strauss @afichter-va and I regrouped. We are okay with the ^ approach recommended by PW and content and IA teams.  

Just to re-emphasize here, what Samara noted above - and which @DanielleThierryUSDSVA and @peggygannon are also aligned on: 

> ... in order to avoid the user profile from becoming a lengthy FAQ, we'll have to work together to pare down the FAQ from what it is now and only include whatever is most relevant and most needed. We don't have a separate FAQ page/section for authenticated tools, and while maybe that may be something we want to consider in the future, it would be out of scope fo this particular project. So, that leaves us putting things contextually in the Connected Apps section of the Profile, and I want to make sure we curate anything that goes on the profile with care.

The above approach is for this release. Longer term:
1/ When there are more 3rd-party apps and the apps info is maintainable via API, let's regroup. That would be the right time to look at creating a directory. 

2/ When there is a directory, that would also be a good time to link out from the authenticated experience to a public directory and more learning/resources type of content, which would be ideal to live in the future learning center that the PW team is kicking off.

Per previous comment - @DanielleThierryUSDSVA @peggygannon will work with @Samara-Strauss 's team and @careyotto to finalize the contextual help information for the authenticated connected apps page.

**Comment by Samara-Strauss** [comment]
All sounds good to me! @DanielleThierryUSDSVA @peggygannon let me know when you're ready to regroup on this. Happy to meet any time.

**Comment by afichter-va** [comment]
@jenniferlee-dsva and @Samara-Strauss: As sated earlier in this issue below, we'll also be adding content on the existing Sign in FAQs page to help guide Veterans to the Connected Accounts section so that this is searchable. 

> Include a few general questions as a “connected apps” section within the existing Sign in FAQs page. This will allow users who aren’t signed in to find this general content via search. In the future, all of this will be great content to move into the VA.gov learning center, which will be designed to house help-related content and other resource-focused content.

Additionally, we'd want to ensure that for the Learning Center content, it can be more robust/extensive and can be linked to easily.

For all content, @careyotto will work closely with the content team to ensure it serves the need for the Lighthouse program.

**Comment by davem-va** [comment]
Thanks all for working through this.

@afichter-va Can we get something in the backlog for an API to provide the required content?

**Comment by FranECross** [comment]
Added to [Slack list ](https://dsva.slack.com/lists/T03FECE8V/F07CF7F1L4R?view_id=View07CRB47R4G)that will be provided to Michelle M for review.

**Comment by FranECross** [comment]
Closing ancient ticket as obsolete.
